doc/tools/coqrst/coqdomain.py

```diff
@@ -895,14 +900,19 @@ def add_coq_output_1(self, repl, node):
         if options['reset']:
             repl.sendone("Reset Initial.")
             repl.sendone("Set Coqtop Exit On Error.")
+            repl.sendone("Set Warnings \"+default\".")
         if options['fail']:
             repl.sendone("Unset Coqtop Exit On Error.")
+        if options['warn']:
+            repl.sendone("Set Warnings \"default\".")
         for sentence in self.split_sentences(node.rawsource):
             pairs.append((sentence, repl.sendone(sentence)))
         if options['abort']:
             repl.sendone("Abort All.")
         if options['fail']:
             repl.sendone("Set Coqtop Exit On Error.")
+        if options['warn']:
+            repl.sendone("Set Warnings \"+default\".")
```

Comment:
Might be cleaner using single quotes to avoid the need for escaping (here and the other Set Warnings).